devel/jsonnet/Portfile

```diff
@@ -54,16 +54,15 @@ if {${name} eq ${subport}} {
 # Name consistency with ${python.branch} and ${python.version} in
 # ${prefix}/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/_resources/port1.0/group/python-1.0.tcl
 
-set python_branches {3.7 3.8 3.9}
+set python_branches {3.9 3.10 3.11 3.12 3.13}
```

Comment:
I assume you verified that there are no dependents of the subports for Python 3.7 and 3.8?